phpstan.neon

```diff
@@ -9,3 +9,7 @@ parameters:
 
     symfony:
         container_xml_path: var/cache/dev/App_KernelDevDebugContainer.xml
+
+    ignoreErrors:
```

Comment:
This should go to baseline file. `phpstan.neon` should be used for project specific errors, while baseline is stuff coming from media site.

Reply:
done in https://github.com/netgen/media-site/commit/a29e13d51ab9f1c3a7985c7a26373d7624477a87